.env.tpl

```diff
@@ -71,3 +71,6 @@ HOST_MINIO_DATA_DIR=$HOST_MINIO_DATA_DIR
 # EPISCANNER
 EPISCANNER_PSQL_URI="postgresql://${PSQL_USER}:${PSQL_PASSWORD}@${PSQL_HOST}:${PSQL_PORT}/${PSQL_DB}"
 EPISCANNER_HOST_DIR=$EPISCANNER_HOST_DIR
+
+# Sentry
+SENTRY_DSN=
```

Comment:
please include the alias for this variable, it's a good practice in case this variable is preset in the environment. It should be: `SENTRY_DSN=$SENTRY_DSN`